app/components/__test__/Button.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { Button, LinkButton } from '../Button';
+
+describe('Button', () => {
+  const btnText = 'Click me';
+  it('renders', () => {
+    const { getByText } = render(
+      <Button size="small" label="Click me" varient="primary" disabled={false} />
+    );
+    expect(getByText(btnText)).toBeInTheDocument();
+  });
+
+  it('renders with different sizes', () => {
+    const { getByText } = render(
+      <Button size="small" label="Click me" varient="primary" disabled={false} />
+    );
+    expect(getByText(btnText)).toHaveClass('py-1 px-2 text-sm gap-1');
+    expect(getByText(btnText).classList.contains('border-transparent')).toBeTruthy();
+    expect(getByText(btnText)).not.toBeDisabled();
+  });
+
+  it('renders with different varients', () => {
+    const handleClick = jest.fn(() => null);
```

Comment:
Can we just use `jest.fn()` here?

Reply:
It doesn't make much difference. Just storing it in variable for reuse

Reply:
I wanted to ask can we just use `handleClick = jest.fn()`? If yes then we don't need to pass the callback function `() => null`